### PR TITLE
updated the readme

### PR DESCRIPTION
**Pull request by jspaaks**
- install instructions for user
- install instructions for developer
- added linter
- changed test command
- removed superfluous text

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -140,15 +132,20 @@ To install matchms, do:
 
   git clone https://github.com/matchms/matchms.git
   cd matchms
-  conda env create --name matchms --file=environment.yml
+  conda env create
   conda activate matchms
-  pip install .
```
Comment (sverhoeven) [inline README.rst:145]: The `conda env create` only installs the matchms dependencies and not matchms itself. 
With `pip install --editable .` you will get the matchms package installed in the Python path.
So I would still do a `pip install --editable .` or a `python setup.py develop`.

This way `matchms` package can be imported independent of in which directory you are currently in.

File: README.rst [hunk 2]
```diff
@@ -140,15 +132,20 @@ To install matchms, do:
...
+.. code-block:: console
+
+  prospector
 
 Run tests (including coverage) with:
 
 .. code-block:: console
 
-  python setup.py test
+  pytest
```
Comment (sverhoeven) [inline README.rst:149]: Nice as `python setup.py --help-commands` lists `test`  command as deprecated.

File: README.rst [hunk 3]
```diff
@@ -88,7 +80,7 @@ Install matchms from PyPI with
 
 .. code-block:: console
 
-  pip install --user matchms
+  conda install matchms --channel nlesc
```
Comment (sverhoeven) [inline README.rst]: Could you change it to ` conda install --channel nlesc matchms`? I would like to see the most important part on the outside of a command.

File: README.rst [hunk 4]
```diff
@@ -73,10 +69,6 @@ https://badgen.net/ to see which other badges are available.)
 
 Vector representation and similarity measure for mass spectrometry data
 
-The project setup is documented in `a separate document <project_setup.rst>`_.
```
Comment (sverhoeven) [inline README.rst:76]: The `project_setup.rst` was generated by the https://github.com/NLeSC/python-template.

Do you plan to delete the `project_setup.rst` file as well or just the link to it?

File: README.rst (hunk 3 above)
Comment (jspaaks) [reply README.rst]: done

File: README.rst (hunk 4 above)
Comment (jspaaks) [reply README.rst:76]: If it's up to me, I'd delete it.

File: README.rst (hunk 1 above)
Comment (jspaaks) [reply README.rst:145]: went with the editable option

File: README.rst (hunk 4 above)
Comment (jspaaks) [reply README.rst:76]: deleted the file as well

**Review by fdiblen — APPROVED** [review]
Great

**Review by sverhoeven — APPROVED** [review]

**Pull request merged**